pkg/logentry/stages/labeldrop.go

```diff
@@ -0,0 +1,61 @@
+package stages
+
+import (
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/mitchellh/mapstructure"
+	"github.com/pkg/errors"
+	"github.com/prometheus/common/model"
+)
+
+const (
+	// ErrEmptyLabelDropStageConfig error returned if config is empty
+	ErrEmptyLabelDropStageConfig = "drop_label stage config cannot be empty"
+)
+
+// LabelDropConfig is a slice of labels to be dropped
+type LabelDropConfig []string
+
+func validateLabelDropConfig(c LabelDropConfig) error {
+	if c == nil || len(c) < 1 {
+		return errors.New(ErrEmptyLabelDropStageConfig)
+	}
+
+	return nil
+}
+
+func newLabelDropStage(logger log.Logger, configs interface{}) (*labelDropStage, error) {
+	cfgs := &LabelDropConfig{}
+	err := mapstructure.Decode(configs, cfgs)
+	if err != nil {
+		return nil, err
+	}
+
+	err = validateLabelDropConfig(*cfgs)
+	if err != nil {
+		return nil, err
+	}
+
+	return &labelDropStage{
+		cfgs:   *cfgs,
+		logger: logger,
+	}, nil
+}
+
+type labelDropStage struct {
+	cfgs   LabelDropConfig
+	logger log.Logger
```

Comment:
I think the logger is unused and can be removed.

Reply:
I removed the logger since as you said it was not being used.